lib/cli/package.json

```diff
@@ -20,9 +20,6 @@
     "type": "git",
     "url": "https://github.com/storybooks/storybook.git"
   },
-  "scripts": {
```

Comment:
Yeah the postinstall was messing with hoisting which I experimented with, When refactoring the cli I will add a opencollective notice during first run or something like it.